### PR TITLE
Astro Colony changed from wine to Linux

### PR DESCRIPTION
**Pull request by Limmek**
# Description

Changed from wine to native linux server.
Since steam has no linux version yet we use [AstroColonyServerScript](https://github.com/FuriosIra/AstroColonyServerScript) & the link in that script to update the server files.

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [ ] You verify that the start command applied does not use a shell script
  * [x] If some script is needed then it is part of a current yolk or a PR to add one
* [x] The egg was exported from the panel

### CODE REVIEW

**Comment by QuintenQVD0** [comment]
Startup can not be 1 script.

**Comment by gOOvER** [comment]
AstroColonie Linux Server is not released yet; only in googledrive for testing; So no need to upate that egg

**Comment by gOOvER** [comment]
AstroColonie Linux Server is not released yet; only in googledrive for testing; So no need to upate that egg

> Startup can not be a script.

i think, i dont take a look into that dir :) . The filename is correct, but he added a .sh :D

File: game_eggs/steamcmd_servers/astro_colony/egg-astro-colony.json [hunk 1]
```diff
@@ -4,50 +4,38 @@
...
... -o AstroColonyLinuxServer.zip\r\n    chmod +x LinuxServer\/AstroColonyServer.sh\r\n    # add steamclient.so to Astro Colony otherwise it wont display on serverlist\r\n    cp -v linux64\/steamclient.so LinuxServer\/AstroColony\/Binaries\/Linux\/steamclient.so\r\nfi\r\n\r\n## set up settings\r\ncreate_server_settings_ini() {\r\n    cat > ServerSettings.ini <<EOF\r\n[\/Script\/AstroColony.EHServerSubsystem]\r\nServerPassword=${SERVER_PASSWORD}\r\nSeed=${SERVER_MAPSEED}\r\nMapName=${SERVER_MAPNAME}\r\nMaxPlayers=${SERVER_MAXPLAYERS}\r\nShouldLoadLatestSavegame=${SERVER_LOADLATESTSAVEGAME}\r\nAdminList=${SERVER_ADMINS} \r\nSharedTechnologies=${SERVER_SHAREDTECHNOLOGIES}\r\nOxygenConsumption=${SERVER_OXYGENCONSUMPTION}\r\nFreeConstruction=${SERVER_FREECONSTRUCTION}\r\nAutosaveInterval=${SERVER_AUTOSAVEINTERVAL}\r\nAutosavesCount=${SERVER_AUTOSAVESCOUNT}\r\nEOF\r\n}\r\n\r\nmkdir -p LinuxServer\/AstroColony\/Saved\/Config\/LinuxServer\r\ncd LinuxServer\/AstroColony\/Saved\/Config\/LinuxServer\r\ncreate_server_settings_ini\r\n\r\n## install end\r\necho \"-----------------------------------------\"\r\necho \"Installation completed...\"\r\necho \"-----------------------------------------\"",
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/astro_colony/egg-astro-colony.json:25]: sry, but you cant use that script to install the server

`https:\/\/raw.githubusercontent.com\/FuriosIra\/AstroColonyServerScript\/main\/InstallAstroColonyLinuxServer.sh`

Did you ever read, how this script works?

**Review by gOOvER — CHANGES_REQUESTED** [review]

**Comment by gOOvER** [comment]
This Whole egg was never tested: He use a script, which is for bare Servers. but not for docker. He dont know, what is installed in the binarie folders. I would close this PR, because there is no official Linux server yet; only a testing Server, which is hosted at Google Drive. I have a working egg, but i dont release it becaus eof googledrive hosting

**Comment by QuintenQVD0** [comment]
We don't allow for script. And if it is still in testing and needs a google drive url. Thne this can be closed until they release ir over the offical steamcmd way.

**Pull request closed without merging**